sandbox/Sandbox/Generated.cs

```diff
@@ -1033,7 +1031,7 @@ public void Serialize(ref MessagePackWriter writer, global::ArrayTestTest value,
             var __MyProperty5__ = default(global::GlobalMyEnum[]);
             var __MyProperty6__ = default(global::QuestMessageBody[]);
 
-            for (int i = 0; i < length; i++)
+            for (var i = 0; i < length; i++)
```

Comment:
Another example of an irrelevant change that makes for very long diffs.
This too, I would reject in its own PR. `var` doesn't add anything here. It's just as short as `int`, but less descriptive.

Reply:
I prefer `var` but I agree with you.